pkg/telemetry/perf_windows.go

```diff
@@ -0,0 +1,27 @@
+package telemetry
+
+import (
+	"runtime"
+	"strconv"
+)
+
+type PerfProfile struct{}
+
+func (p *PerfProfile) GetMemoryUsage() map[string]string {
+	var m runtime.MemStats
+	runtime.ReadMemStats(&m)
+	props := map[string]string{
+		allocatedmem: strconv.FormatUint(bToMb(m.Alloc), 10),
+		sysmem:       strconv.FormatUint(bToMb(m.Sys), 10),
+		goroutines:   strconv.Itoa(runtime.NumGoroutine()),
+	}
+	return props
+}
+
+func NewPerfProfile() (*PerfProfile, error) {
+	return &PerfProfile{}, nil
+}
+
+func (p *PerfProfile) GetCPUUsage() (map[string]string, error) { //nolint unnamed results are fine
+	return make(map[string]string), nil
```

Comment:
Any reason why we don't have CPU usage data on windows😞?

Reply:
this syscall getrusprofile doesn't exist over there, can leave it open ended for windows in the future